packages/amplify-ui-components/src/components/amplify-select/amplify-select.tsx

```diff
@@ -0,0 +1,31 @@
+import { Component, Prop, h } from '@stencil/core';
+import { select } from './amplify-select.style';
+import { styleNuker } from '../../common/helpers';
+import { AMPLIFY_UI_SELECT } from '../../common/constants';
+import { SelectOptions } from './amplify-select-interface';
+import countryDialCodes from '../../common/country-dial-codes';
+
+@Component({
+  tag: 'amplify-select',
+})
+export class AmplifySelect {
+  /** (Optional) Overrides default styling */
+  @Prop() styleOverride: boolean = false;
+  /** Must be an Array of Objects with an Object shape of {label: string, value: string|number} */
```

Comment:
Nit: first and foremost put the purpose of the prop
```suggestion
  /** The options of the select input.  Must be an Array of Objects with an Object shape of {label: string, value: string|number} */
```

Reply:
adjusted